source/hardware/controllers/native_instruments_traktor_kontrol_s3.rst

```diff
@@ -212,7 +212,8 @@ Effect Chain Mode
 In Effect Chain Mode, the :hwlabel:`FILTER ENABLE` buttons change color to match the selected FX button.
 The lights will be dim if the effect is disabled, and bright if it is enabled.
 Tap the :hwlabel:`FILTER ENABLE` button to enable or disable the effect.
-Turn the filter knobs to adjust the meta knob for each effect.
+Turn the first three filter knobs to adjust the meta knob for each effect.
+The last filter knob adjusts the meta knob for the whole chain.
```

Comment:
meta knob for the whole chain? There is no such thing. Do you mean the mix knob or the superknob?